src/lib/heat.ml

```diff
@@ -1,4 +1,25 @@
 
+(* This file is free software, part of FTW. See file "LICENSE" for more information *)
+
+(* Type definitions *)
```

Comment:
why call it `heat.ml` ?

Reply:
A "heat" is the english technical term for what we'd call "poule" in french (see [this](https://support.rosterathletics.com/en/support/solutions/articles/44001356515-words-and-terms#:~:text=HEATS%20When%20an%20event%20has,%22Timed%20Final%20Heats%22) for instance) and what was (very mistakenly) called "pool" in the old `ftw`.